README.md

```diff
@@ -86,6 +86,7 @@ Parameters for the app are stored in [environment variables](http://man7.org/lin
 | `VUE_APP_CONF_API_URL` | URL from which the app can retrieve launch context, i.e. launching as a confidential client | ''
 | `VUE_APP_PROJECT_ID` | Project ID that uniquely identifies the project.  The app will display project specific logo and theme based on the ID | 'DCW'
 | `VUE_APP_DASHBOARD_URL` | URL for returning to FEMR dashboard app | Example: `https://dashboard.acc.dev.cosri.cirg.washington.edu`
+| `VUE_APP_ALLOW_SKIPPING_QUESTIONNAIRE`| Specify `false` if the feature to allow skipping questionnaire is to be disabled | true
```

Comment:
@achen2401  doesn't the "| true" at the end of this comment line indicate that this is default true?

Reply:
@mcjustin hmm, okay, "defaulting" to true means that without the `VUE_APP_ALLOW_SKIPPING_QUESTIONNAIRE` being set, the app should allow skipping questionnaire, is that right?  If so, [here](https://github.com/uwcirg/asbi-screening-app/blob/7aec78a390805afcfb08386e37b48f560c886885/src/components/Survey.vue#L111)  would be come `String(getEnv("VUE_APP_ALLOW_SKIPPING_QUESTIONNAIRE")) === "false" ? false : true` ?

Reply:
I was just trying to understand what the comment means... I think it's fine as-is functionally, but it would be good if the comments indicated what the defaults are.